lib/charms/tls_certificates_interface/v1/tls_certificates.py

```diff
@@ -126,6 +126,7 @@ def _on_certificate_revocation_request(self, event: CertificateRevocationRequest
 from charms.tls_certificates_interface.v1.tls_certificates import (
```

Comment:
1. okay! I will need some help with this I am not the most git savy and do not yet know how to remove a commit from git

2. Yes they are versioned. The changes made to `lib/charms/tls_certificates_interface/v1/tls_certificates.py` were made with `fetch-lib` so these are the most updated changes. 

No these changes shouldn't be made with the `TLS-certificates-operator` charm. (Two reasons)
- these changes were already made and fetched with `fetch-lib` 
- the library for `tls_certificates_interface` **is not the ** `TLS-certificates-operator` operator

Reply:
Sorry, when I wrote `tls-certificates-operator`, I meant `tls-certificates-interface` repo

Anyhow, I wanted to double check that no code review was needed for that bit. I'd still separate the two commits (let's sync later to show you how this can be achieved, but yes, in general we need to agree how to cope with changes in charm libraries when it comes to commits/PR)